package.json

```diff
@@ -227,6 +229,7 @@
     "semver": "^5.7.0",
     "shell-escape": "^0.2.0",
     "split": "^1.0.1",
+    "ssh2": "^0.8.5",
```

Comment:
Not sure how important, but this PR was parked for a little while and there are newer versions of ssh2 (0.8.7) and types/ssh2 (0.5.39).